doc/user-manual/language/rewriting.lagda.rst

```diff
@@ -9,6 +9,12 @@ Rewriting
 Rewrite rules allow you to extend Agda's evaluation relation with new
 computation rules.
 
+Rules are safe to use with ```Agda.Builtin.Equality``
+if :ref:`--confluence-check <confluence-check>` is enabled.
+Confluent but non-terminating rewrite rules can not break consistency,
+unlike to non-terminating functions.
+Those results was proved `here <https://hal.science/hal-02901011v2/document>`_.
```

Comment:
Grammar: was -> were

> Confluent but non-terminating rewrite rules can not break consistency,

Please add a pointer to _where_ in this paper this result was proven.  Skimming abstract and introduction, I do not find such a statement.  If it is buried somewhere in the paper, there should be a pointer where to find it.

Reply:
I believe section 3 states that:

"
Rewrite rules should only rewrite applications of ‘fresh’ (i.e., postulated) symbols, not preexisting symbols like → or Box.
• The rewriting system defined by the rewrite rules together with the standard reduction rules
of type theory should be confluent.
From these natural restrictions, we derive injectivity of type constructors and hence re-establish
subject reduction."

Reply:
> For one, if you add new constants, you can already become inconsistent:

Yes. But in your example `any` postulate already makes type theory inconsistent. So rewrite rules themselves cannot break consistency.

Reply:
@jespercockx Am I rephrasing result on rewriting safety correctly or missing something?

Should I add something about "Rewrite rules should only rewrite applications of ‘fresh’ (i.e., postulated) symbols? Is this checked by `--confluence-check` as well or depends on user?

Reply:
The Agda implementation of confluence checking actually converts clauses of definitions by pattern matching to rewrite rules, so the restriction that rewrite rules should only be applied to postulates does not apply here.

Reply:
@jespercockx : I you are happy with the formulations, I let you merge this.